CONTRIBUTING.md

```diff
@@ -69,69 +69,56 @@ please adhere to the following contributing guidelines.
     skipAfter
 ```
 
+## Variables naming conventions
+
+* Variable names are lowercase using chars from `[a-z0-9_]`
+* To somewhat reflect the fact that the syntax for variable usage is different when you define it (using setvar) and when you use it, we propose the following visual distinction:
+  * Capital letters for collection, colon as separator, variable name. E.g: `SecRule TX:foo_bar_variable`
```

Comment:
In your description, setvar is first, then in the example, SecRule is first. Please align it.

Reply:
Done.